src/login.tsx

```diff
@@ -57,13 +58,13 @@ export default function LoginPage() {
       id="login-page"
       data-testid='login'
       component="form"
+      noValidate
+      autoComplete="off"
```

Comment:
Should we enable autocomplete for login?

Reply:
Changed this before I saw this comment. See my response above!